src/gateway/get-started/services-and-routes.md

```diff
@@ -0,0 +1,455 @@
+   ```text
+   {
+     "host": "mockbin.org",
+     "name": "example_service",
+     "enabled": true,
+     "connect_timeout": 60000,
+     "read_timeout": 60000,
+     "retries": 5,
+     "protocol": "http",
+     "path": null,
+     "port": 80,
+     "tags": null,
+     "client_certificate": null,
+     "tls_verify": null,
+     "created_at": 1661346938,
+     "updated_at": 1661346938,
+     "tls_verify_depth": null,
+     "id": "3b2be74e-335b-4f25-9f08-6c41b4720315",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
+   ```
+   
+   Fields that are not explicitly provided in the create request are automatically given 
+   a default value based on the current {{site.base_gateway}} configuration.  
+   The [Admin API documentation](/gateway/latest/admin-api/#service-object) has the 
+   full specification for managing service objects.
+
+1. **Viewing service configuration**
+
+   When you create a service, {{site.base_gateway}} assigns it a unique `id` as shown in the response above. 
+   The `id` field, or the name provided when creating the service, can be used to identify the service 
+   in subsequent requests. This is the service URL and takes the form of `/services/{service name or id}`.
+
+   To view the current state of a service, make a `GET` request on the service
+   URL.
+
+   ```sh
+   curl -X GET http://localhost:8001/services/example_service
+   ```
+   
+   The response body contains the current configuration of your service: 
+   
+   ```text
+   {
+     "host": "mockbin.org",
+     "name": "example_service",
+     "enabled": true,
+     "connect_timeout": 60000,
+     "read_timeout": 60000,
+     "retries": 5,
+     "protocol": "http",
+     "path": null,
+     "port": 80,
+     "tags": null,
+     "client_certificate": null,
+     "tls_verify": null,
+     "created_at": 1661290122,
+     "updated_at": 1661290122,
+     "tls_verify_depth": null,
+     "id": "c1e98b2b-6e77-476c-82ca-a5f1fb877e07",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
+   ```
+
+1. **Updating services**
+
+   Existing service configurations can be updated dynamically by sending a `PATCH`
+   request to the service URL. 
+
+   To dynamically set the service retries from `5` to `6`, send this `PATCH` request:
+
+   ```sh
+   curl --request PATCH \
+     --url localhost:8001/services/example_service \
+     --data retries=6
+   ```
+   
+   The response body displays the updated change:
+   
+   ```sh
+   {
+     "host": "mockbin.org",
+     "name": "example_service",
+     "enabled": true,
+     "connect_timeout": 60000,
+     "read_timeout": 60000,
+     "retries": 6,
+     "protocol": "http",
+     "path": null,
+     "port": 80,
+     "tags": null,
+     "client_certificate": null,
+     "tls_verify": null,
+     "created_at": 1661290122,
+     "updated_at": 1661292428,
+     "tls_verify_depth": null,
+     "id": "c1e98b2b-6e77-476c-82ca-a5f1fb877e07",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
```

Comment:
```suggestion
   {
     "retries": 6,
   }
```

How do you feel about something like this focusing the reader into the change you want them to see?

Reply:
I can make a change like this, but I suggest we indicate it's a snippet of the full response.  I've done this elsewhere with:

```
...
"retries": 6,
...
```

Reply:
Please see latest